packaging/rpm/cvmfs-universal.spec

```diff
@@ -35,7 +35,7 @@ Summary: CernVM File System
 Name: cvmfs
 Version: 2.6.0
 Release: 1%{?dist}
-Source0: https://ecsft.cern.ch/dist/cvmfs/%{name}-%{version}.tar.gz
+Source0: https://ecsft.cern.ch/dist/cvmfs/%{name}-%{version}/source.tar.gz
```

Comment:
The name should stay as before, i.e. https://ecsft.cern.ch/dist/cvmfs/%{name}-%{version}/%{name}-%{version}.tar.gz